From 3e7e7a886006376ba68e959addaea1eabbb37482 Mon Sep 17 00:00:00 2001
From: Christian Hesse <mail@eworm.de>
Date: Mon, 2 Apr 2018 23:09:01 +0200
Subject: [PATCH] git: update to v2.17.0

Update to git version v2.17.0. Required changes:

* The function 'typename' has been renamed to 'type_name'
  (upstream commit debca9d2fe784193dc2d9f98b5edac605ddfefbb)
---
 Makefile    | 2 +-
 git         | 2 +-
 ui-shared.c | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git ui-shared.c ui-shared.c
index 9d8f66b..ce806f6 100644
--- ui-shared.c
+++ ui-shared.c
@@ -545,7 +545,7 @@ void cgit_object_link(struct object *obj)
 		page = "tag";
 	else
 		page = "blob";
-	name = fmt("%s %s...", typename(obj->type), shortrev);
+	name = fmt("%s %s...", type_name(obj->type), shortrev);
 	reporevlink(page, name, NULL, NULL, ctx.qry.head, fullrev, NULL);
 }
 
-- 
2.17.0

From 3e7e7a886006376ba68e959addaea1eabbb37482 Mon Sep 17 00:00:00 2001
From: Christian Hesse <mail@eworm.de>
Date: Mon, 2 Apr 2018 23:09:01 +0200
Subject: [PATCH] git: update to v2.17.0

Update to git version v2.17.0. Required changes:

* The function 'typename' has been renamed to 'type_name'
  (upstream commit debca9d2fe784193dc2d9f98b5edac605ddfefbb)
---
 Makefile    | 2 +-
 git         | 2 +-
 ui-shared.c | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git ui-shared.c ui-shared.c
index 9d8f66b..ce806f6 100644
--- ui-shared.c
+++ ui-shared.c
@@ -545,7 +545,7 @@ void cgit_object_link(struct object *obj)
 		page = "tag";
 	else
 		page = "blob";
-	name = fmt("%s %s...", typename(obj->type), shortrev);
+	name = fmt("%s %s...", type_name(obj->type), shortrev);
 	reporevlink(page, name, NULL, NULL, ctx.qry.head, fullrev, NULL);
 }
 
-- 
2.17.0

